daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

```diff
@@ -0,0 +1,61 @@
+---
+type: docs
+title: "WASM"
+linkTitle: "WASM"
+description: "Use WASM middleware in your HTTP pipeline"
+aliases:
+- /developing-applications/middleware/supported-middleware/middleware-wasm/
+---
+
+The WASM [HTTP middleware]({{< ref middleware.md >}}) component enables you to use a WASM module to handle requests and responses.
+Using WASM modules allow developers to write middleware components in any WASM supported language and extend Dapr using external files that are not pre-compiled into the `daprd` binary.
+
+WASM modules are loaded from a filesystem path. On Kubernetes, see [mounting volumes to the Dapr sidecar]({{> kubernetes-volume-mounts.md >}}) to configure a filesystem mount that can contain WASM modules.
+
+## Component format
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: wasm
+spec:
+  type: middleware.http.wasm.basic
+  version: v1
+  metadata:
+  - name: path
+    value: "./hello.wasm"
+  - name: runtime
+    value: "wazero"
+```
+
+## Spec metadata fields
+
+| Field | Details | Example |
+|-------|---------|---------|
+| path | The path to the WASM binary | "./hello.wasm" |
```

Comment:
This is relative to where the configuration file is correct? Probably should say so.

Reply:
Done